### PR TITLE
Add permission to view throughput report

### PR DESCRIPTION
**Pull request by kluchrj**
Also fix missing comma in permissions for limitedaccessuserprofile, which was causing errors when migrating.

### CODE REVIEW

File: timepiece/crm/models.py [hunk 1]
```diff
@@ -315,8 +317,10 @@ class PaidTimeOffRequest(models.Model):
 
     class Meta:
         ordering = ('user_profile', '-pto_start_date',)
-        permissions = (("can_approve_pto_requests", "Can approve PTO requests"),
-                       ("can_process_pto_requests", "Can payroll process PTO requests"), )
+        permissions = (
+            ("can_approve_pto_requests", "Can approve PTO requests"),
```
Comment (dannybrowne86) [inline timepiece/crm/models.py]: Can you remove the `can_` here and make it singular?  I know this was from me originally, but it should just be `approve_pto_request` to follow standard practice.  Can you do that throughout this PR, where I may have messed it up in the past?
We will want to do a `git grep -n can_approve_pto_requests` after making the change to see if there are other places in code we reference that permission.

File: timepiece/crm/models.py [hunk 2]
```diff
@@ -1250,7 +1255,9 @@ def __unicode__(self):
 
     class Meta:
         ordering = ('due_date', 'name')
-        permissions = (('approve_milestone', 'Can approve milestone'),)
+        permissions = (
+            ('approve_milestone', 'Can approve milestone'),
```
Comment (dannybrowne86) [inline timepiece/crm/models.py:1259]: This is a good example of a permission.

**Comment by kluchrj** [comment]
After changing permissions around, there is an error when going to the crm users page.

`Reverse for 'view_user' with arguments '(-1,)' and keyword arguments '{}' not found.`

~~This error is caused by the anonymous user, which on the live site has and id of 85. I'm not sure if it's related to this PR, but it appeared after migrating.~~

Edit:
I looked into it more and the "AnonymousUser" is from guardian. I don't think it has a user object in the live site, but after changing the permissions around it creates one.

File: timepiece/crm/models.py (hunk 1 above)
Comment (kluchrj) [inline timepiece/crm/models.py]: Done.

**Pull request merged**